### DISCUSSION TITLE
Remotely read only partitioned parquet files on S3 with R

### DISCUSSION THREAD

**Discussion by baslat** [duckdb/duckdb#9225] [Q&A]
Hello,

I saw [this post ](https://r.iresmi.net/posts/2023/fast_remote_parquet/index.html) that demonstrates "querying a remote [parquet](https://parquet.apache.org/) file via HTTP with [DuckDB](https://duckdb.org/)". It suggests that the file does **not** need to be downloaded when read this way. I would like to know, is this possible for partitioned parquet files on (private) S3, and if so, how? 

So far everything I have tried results in the files being copied to my machine, so any help would be appreciated.

Thanks!

**Comment by Mause** [comment]
If the web server supports range downloads, then the full file does not need to be downloaded. Otherwise, DuckDB will have to download it in full

**Reply by baslat** [reply]
Thanks; how do I check if the web server supports range downloads? And assuming it does, what would the R syntax look like?
